### ISSUE TITLE
Try to start a new training

### ISSUE THREAD

**Issue by SeanChenxy** [korrawe/grasping_field_demo#5]
Hi korrawe. Thanks for sharing this nice work. I am very interested in this repo, but I find the training procedure is hard to follow. Could you release more detailed instructions, such as how to prepare obman data, how to select training parameters? 
Looking for your reply. Thanks in advance.

**Comment by anilesec** [comment]
Thanks, @SeanChenxy  for creating this issue.  I am also looking for this information. 
@korrawe Thanks in Advance!

**Comment by anilesec** [comment]
@SeanChenxy Did you manage to train?
